src/style.css

```diff
@@ -1 +1,37 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  gap: 4px;
```

Comment:
![image](https://github.com/mate-academy/layout_stars/assets/80700247/8368f5b3-bde2-46fc-a752-a299f7dc36a5)

Use margin-right instead and don't forget to remove margin-right from the last stars in a row (you can use :last-child rule)